trunk/SUAVE/Components/Energy/Converters/Motor_Lo_Fid.py

```diff
@@ -100,6 +102,7 @@ def omega(self,conditions):
 
         # Omega
         omega1 = (Kv*v)/2. + (Kv*(Res*Res*io*io - 2.*Res*etam*io*v - 2.*Res*io*v + etam*etam*v*v - 2.*etam*v*v + v*v)**(1./2.))/2. - (Kv*Res*io)/2. + (Kv*etam*v)/2.
+        omega1[np.isnan(omega1)] = 0.0
```

Comment:
Could you add a comment saying what situation creates a NaN?

Reply:
Done